specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,227 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-07-01-preview",
+    "title": "SymsClient"
```

Comment:
>SymsClient [](http://example.com/codeflow?start=14&length=10)

Would it better to call it MetadataClient? We usually don't use abbreviation for client name.